tests/unit/data_tests.py

```diff
@@ -7,40 +7,48 @@
 import decimal
 import unittest
 from collections import OrderedDict
+from pika.compat import PY2, PY3
 
 from pika import data, exceptions
 from pika.compat import long
 
 
 class DataTests(unittest.TestCase):
 
-    FIELD_TBL_ENCODED = (
-        b'\x00\x00\x00\xcb'
-        b'\x05arrayA\x00\x00\x00\x0fI\x00\x00\x00\x01I'
-        b'\x00\x00\x00\x02I\x00\x00\x00\x03'
-        b'\x07boolvalt\x01'
-        b'\x07decimalD\x02\x00\x00\x01:'
-        b'\x0bdecimal_tooD\x00\x00\x00\x00d'
-        b'\x07dictvalF\x00\x00\x00\x0c\x03fooS\x00\x00\x00\x03bar'
-        b'\x06intvalI\x00\x00\x00\x01'
-        b'\x06bigint\x6c\x00\x00\x00\x00\x9a\x7e\xc8\x00'
-        b'\x07longval\x6c\x00\x00\x00\x00\x36\x65\x26\x55'
-        b'\x04nullV'
-        b'\x06strvalS\x00\x00\x00\x04Test'
-        b'\x0ctimestampvalT\x00\x00\x00\x00Ec)\x92'
-        b'\x07unicodeS\x00\x00\x00\x08utf8=\xe2\x9c\x93')
+    FIELD_TBL_ENCODED = b"".join((
```

Comment:
Although more elegant, changing all the existing lines makes it difficult to review.